index.js

```diff
@@ -58,6 +58,8 @@ module.exports = function(source) {
 
   var debug = query.debug;
 
+  var ignorePartials = !!query.ignorePartials
```

Comment:
I don't think this line is really necessary if you just use `query.ignorePartials` directly below.